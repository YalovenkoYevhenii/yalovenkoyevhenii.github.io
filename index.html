<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>ВРЕМЯ:ВИСКИ</title>
    <link rel="stylesheet" href="css/fonts.css">
    <link rel="stylesheet" href="css/iconFont.css">
    <link rel="stylesheet" href="/css/slick.css">
    <link rel="stylesheet" href="css/style.css">
</head>
<body>
    <header class="header">
        <div class="container">
            <div class="wrapper">
                <div class="header-burger">
                    <span class="header-burger__inner">
                    </span>
                </div>
                <div class="logo">
                    <a href="#" class="logo__link">
                        <div class="logo__sign">ВРЕМЯ:<span class="thin">ВИСКИ</span></div>
                    </a>
                    <div class="logo__slogan">Время пить виски</div>
                </div>
                <div class="phone-block">
                    <span class="icon-phone">
                    </span>
                    <a href="tel:+380671234567" class="telephone">+38 (067) 123 45 67</a>
                </div>
                <form class="search-line__form">
                    <input type="text" class="input" placeholder="Что вы ищите?">
                    <button type="submit" class="button"><span class="icon-magnifier"></span></button>
                </form>
                
                <div class="user-product-collection">
                    <div class="favorites">
                        <span class="icon-heart"></span>
                    </div>
                    <div class="shopping-cart">
                        <span class="icon-shoppingCart"></span>
                    </div>
                </div>
            
            
                <div class="user-block">
                    <span class="icon-userCircle"></span>
                    <div class="user-block__name">Иван Иванов</div>
                    <span class="icon-caretDown"></span>
                </div>
            
            </div> 
            
        </div>
    </header>
    <div class="mobile-menu">
        <div class="mobile-menu-user-block">
            <div class="container">
                <div class="mobile-menu-user-main">
                    <span class="icon-userCircle icon-userCircle_mobile"></span>
                    <div class="mobile-menu-user-main__name">Иван Иванов</div>
                    <span class="icon-caretDown icon-caretDown_mobile"></span>
                    
                </div>
                <div class="personal-settings">
                    <a class="personal-settings__info">Личные данные</a>
                    <a class="personal-settings__orders">Мои заказы<span class="order-count"> (2)</span></a>
                    <a class="personal-settings__favorites">Избранное<span class="favorites-count"> (3)</span></a>
                    <a class="personal-settings__account-exit">Выйти из аккаунта</a>        
                </div>
            </div>
        </div>
        <div class="mobile-menu-customer-info">
            <div class="container">
                <div class="btn-mobile">
                    <span class="btn-mobile__name">Перейти в каталог</span>
                    <img src="assets/icons/ArrowRight.svg" alt="arrow" class="btn-mobile__icon">
                </div>
                <a class="mobile-menu-customer-info__item">Доставка и оплата</a>
                <a class="mobile-menu-customer-info__item">Возврат и обмен</a>
                <a class="mobile-menu-customer-info__item">Контакты</a>
                <a class="mobile-menu-customer-info__item">О нас</a>
            </div>
        </div>
        <div class="mobile-menu-footer-contacts">
            <div class="container">
                <div class="mobile-menu-footer-contacts__phone-block">
                    <img src="assets/icons/Phone.svg" alt="phone-icon" class="phone-block__icon">
                    <a href="tel:+380671234567" class="telephone">+38 (067) 123 45 67</a>
                </div>
                <div class="mobile-menu-footer-contacts__mail-block">
                    <img src="assets/icons/EnvelopeSimple.svg" alt="envelope-icon" class="mail-block__icon">
                    <a href="mailto:whickey@mail.com" class="email">whickey@mail.com</a>
                </div>
                <div class="mobile-menu-footer-contacts__social-media-icons">
                    <a href="#" class="social-media-icon">
                        <img src="assets/icons/instagramm.svg" alt="instagram">
                    </a>
                    <a href="#" class="social-media-icon">
                        <img src="assets/icons/telegramm.svg" alt="telegram">
                    </a>
                    <a href="#" class="social-media-icon">
                        <img src="assets/icons/facebook.svg" alt="facebook">
                    </a>
                </div>
            </div>
        </div>
            
    </div>
    
    <div class="subheader">
        <div class="container">
            <nav class="product-types">
                <div class="product-type__item">
                    Шотландский<br>виски
                </div>
                <div class="product-type__item">
                    Виски разных<br>стран
                </div>
                <div class="product-type__item">
                    Ромы разных<br>стран
                </div>
                <div class="product-type__item">
                    Вино,<br>вермуты
                </div>
                <div class="product-type__item">
                    Прочий<br>алкоголь
                </div>
                <div class="product-type__item">
                    Пиво,<br>напитки
                </div>
                <div class="product-type__item">
                    Кофе,<br>продукты
                </div>
                <div class="product-type__item">
                    Подарочный наборы,<br>аксессуары
                </div>
                <div class="product-type__item">
                    <img class="icon-fire" src="assets/icons/fire.svg" alt="fire_sign">
                    Новинки<br>и Акции
                </div>
            </nav>
        </div>
    </div>
    

    <div class="overlay"></div>
    
    <div class="megamenu-with-buttons">
        <div class="megamenu__content">
            <div class="megamenu__column">
                <div class="btn_megamenu">Вид Напитка 1</div>
                <div class="btn_megamenu">Вид Напитка 2</div>
                <div class="btn_megamenu">Вид Напитка 3</div>
            </div>
            <div class="megamenu__column">
                <div class="megamenu__heading">
                    Тип
                </div>
                <a href="#" class="megamenu_item">
                    Виски номер 1
                </a>
                <a href="#" class="megamenu_item">
                    Виски номер 2
                </a>
                <a href="#" class="megamenu_item">
                    Виски номер 3
                </a>
                <a href="#" class="megamenu_item">
                    Виски номер 4
                </a>
                <a href="#" class="megamenu_item">
                    Виски номер 5
                </a>
            </div>
            <div class="megamenu__column">
                <div class="megamenu__heading">
                    Страна
                </div>
                <a href="#" class="megamenu_item">
                    Страна номер 1
                </a>
                <a href="#" class="megamenu_item">
                    Страна номер 2
                </a>
                <a href="#" class="megamenu_item">
                    Страна номер 3
                </a>
                <a href="#" class="megamenu_item">
                    Страна номер 4
                </a>
                <a href="#" class="megamenu_item">
                    Страна номер 5
                </a>
            </div>
            <div class="megamenu__column">
                <div class="megamenu__heading">
                    Бренды
                </div>
                <a href="#" class="megamenu_item">
                    Бренд номер 1 
                </a>
                <a href="#" class="megamenu_item">
                    Бренд номер 2
                </a>
                <a href="#" class="megamenu_item">
                    Бренд номер 3
                </a>
                <a href="#" class="megamenu_item">
                    Бренд номер 4
                </a>
                <a href="#" class="megamenu_item">
                    Бренд номер 5
                </a>
            </div>
            <div class="megamenu__column">
                <a href="#" class="megamenu-image-block">
                    <span class="megamenu-image-block__image-name">
                        Пиво по акции
                    </span>
                    <img src="/assets/img/beer_banner_megamenu.png" alt="product_photo" class="megamenu-image image_hover-effect">
                </a>
            </div>
        </div>
    </div>

    <div class="megamenu-without-buttons">
        <div class="megamenu__content">
            <div class="megamenu__column">
                <div class="megamenu__heading">
                    Тип
                </div>
                <a href="#" class="megamenu_item">
                    Виски номер 1
                </a>
                <a href="#" class="megamenu_item">
                    Виски номер 2
                </a>
                <a href="#" class="megamenu_item">
                    Виски номер 3
                </a>
                <a href="#" class="megamenu_item">
                    Виски номер 4
                </a>
                <a href="#" class="megamenu_item">
                    Виски номер 5
                </a>
            </div>
            <div class="megamenu__column">
                <div class="megamenu__heading">
                    Выдержка
                </div>
                <a href="#" class="megamenu_item">
                    значение 1
                </a>
                <a href="#" class="megamenu_item">
                    значение 2
                </a>
                <a href="#" class="megamenu_item">
                    значение 3
                </a>
                <a href="#" class="megamenu_item">
                    значение 4
                </a>
                <a href="#" class="megamenu_item">
                    значение 5
                </a>
            </div>
            <div class="megamenu__column">
                <div class="megamenu__heading">
                    Страна
                </div>
                <a href="#" class="megamenu_item">
                    Страна номер 1
                </a>
                <a href="#" class="megamenu_item">
                    Страна номер 2
                </a>
                <a href="#" class="megamenu_item">
                    Страна номер 3
                </a>
                <a href="#" class="megamenu_item">
                    Страна номер 4
                </a>
                <a href="#" class="megamenu_item">
                    Страна номер 5
                </a>
            </div>
            <div class="megamenu__column">
                <div class="megamenu__heading">
                    Бренды
                </div>
                <a href="#" class="megamenu_item">
                    Бренд номер 1 
                </a>
                <a href="#" class="megamenu_item">
                    Бренд номер 2
                </a>
                <a href="#" class="megamenu_item">
                    Бренд номер 3
                </a>
                <a href="#" class="megamenu_item">
                    Бренд номер 4
                </a>
                <a href="#" class="megamenu_item">
                    Бренд номер 5
                </a>
            </div>
            <div class="megamenu__column">
                <a href="#" class="megamenu-image-block">
                    <span class="megamenu-image-block__image-name">
                        Пиво по акции
                    </span>
                    <img src="/assets/img/beer_banner_megamenu.png" alt="product_photo" class="megamenu-image image_hover-effect">
                </a>
            </div>
        </div>
    </div>
    
    <div class="container">        
        <form class="search-line__form-mobile">
            <input type="text" class="input" placeholder="Что вы ищите?">
            <button type="submit" class="button">
                <span class="icon-magnifier"></span>
            </button>
        </form>
    </div>

    <section class="slider-section">
        <div class="container">
            <div class="slider">
                <div class="slider__slide">
                    <picture>
                        <source media="(max-width:576px)" srcset="assets/img/mobile_firstScreenSlider1.png">
                        <img src="assets/img/firstScreenSlider1.png" alt="slide1" class="slider__image image_hover-effect">
                    </picture> 
                    <div class="slider__message"> 
                        <span class="slider__message-type">Спецпредложение</span>
                        <h1 class="message">
                            Виски <span class="product">Jack Daniels</span>
                            в ассортименте
                        </h1>
                        <button class="btn">Посмотреть</button>
                    </div>
                </div>
                <div class="slider__slide">
                    <picture>
                        <source media="(max-width:576px)" srcset="assets/img/mobile_firstScreenSlider1.png">
                        <img src="assets/img/firstScreenSlider1.png" alt="slide1" class="slider__image image_hover-effect">
                    </picture>
                    <div class="slider__message"> 
                        <span class="slider__message-type">Спецпредложение</span>
                        <h1 class="message">
                            Виски <span class="product">Jack Daniels</span>
                            в ассортименте
                        </h1>
                        <button class="btn">Посмотреть</button>
                    </div>
                </div>
                <div class="slider__slide">
                    <picture>
                        <source media="(max-width:576px)" srcset="assets/img/mobile_firstScreenSlider1.png">
                        <img src="assets/img/firstScreenSlider1.png" alt="slide1" class="slider__image image_hover-effect">
                    </picture>
                    <div class="slider__message"> 
                        <span class="slider__message-type">Спецпредложение</span>
                        <h1 class="message">
                            Виски <span class="product">Jack Daniels</span>
                            в ассортименте
                        </h1>
                        <button class="btn">Посмотреть</button>
                    </div>
                </div>
                <div class="slider__slide">
                    <picture>
                        <source media="(max-width:576px)" srcset="assets/img/mobile_firstScreenSlider1.png">
                        <img src="assets/img/firstScreenSlider1.png" alt="slide1" class="slider__image image_hover-effect">
                    </picture>
                    <div class="slider__message"> 
                        <span class="slider__message-type">Спецпредложение</span>
                        <h1 class="message">
                            Виски <span class="product">Jack Daniels</span>
                            в ассортименте
                        </h1>
                        <button class="btn">Посмотреть</button>
                    </div>
                </div>
            </div>
        </div>

    </section>
    <section class="categories">
        <div class="container">
            <h2 class="section-heading">
                Популярные категории
            </h2>
            <div class="categories-wrapper">
                <a class="image-block">
                    <span class="image-block__name">Коньяк</span>
                    <button class="btn_arrow">
                        <span class="icon-caretRight"></span>
                    </button>
                    <picture>
                        <source media="(max-width:680px)" srcset="assets/img/mobile_2nd_section_1.png">
                        <img src="assets/img/2nd_section_1.png" alt="image_1" class="image image_hover-effect i1">
                    </picture>
                </a>
                <a class="image-block">
                    <span class="image-block__name">Джин</span>
                    <button class="btn_arrow">
                        <span class="icon-caretRight"></span>
                    </button>
                    <picture>
                        <source media="(max-width:680px)" srcset="assets/img/mobile_2nd_section_2.png">
                        <img src="assets/img/2nd_section_2.png" alt="image_2" class="image image_hover-effect i2">
                    </picture>
                </a>
                <a class="image-block">
                    <span class="image-block__name">Водка</span>
                    <button class="btn_arrow">
                        <span class="icon-caretRight"></span>
                    </button>
                    <picture>
                        <source media="(max-width:680px)" srcset="assets/img/mobile_2nd_section_3.png">
                        <img src="assets/img/2nd_section_3.png" alt="image_3" class="image image_hover-effect i3">
                    </picture>
                </a>
                <a class="image-block">
                    <span class="image-block__name">Виски</span>
                    <button class="btn_arrow">
                        <span class="icon-caretRight"></span>
                    </button>
                    <picture>
                        <source media="(max-width:680px)" srcset="assets/img/mobile_2nd_section_4.png">
                        <img src="assets/img/2nd_section_4.png" alt="image_4" class="image image_hover-effect i4">
                    </picture>
                </a>
                <a class="image-block">
                    <span class="image-block__name">Ром</span>
                    <button class="btn_arrow">
                        <span class="icon-caretRight"></span>
                    </button>
                    <picture>
                        <source media="(max-width:680px)" srcset="assets/img/mobile_2nd_section_5.png">
                        <img src="assets/img/2nd_section_5.png" alt="image_5" class="image image_hover-effect i5">
                    </picture>
                </a>
                <a class="image-block">
                    <span class="image-block__name">Вино</span>
                    <button class="btn_arrow">
                        <span class="icon-caretRight"></span>
                    </button>
                    <picture>
                        <source media="(max-width:680px)" srcset="assets/img/mobile_2nd_section_6.png">
                        <img src="assets/img/2nd_section_6.png" alt="image_6" class="image image_hover-effect i6">
                    </picture>
                </a>
            </div>
        </div>
    </section>


    

    
    
    <script  src="/js/slider/jquery-3.6.0.min.js"></script>
    <script  src="/js/slider/slick.min.js"></script>
    <script  src="/js/slider/slider.js"></script>
</body>
</html>