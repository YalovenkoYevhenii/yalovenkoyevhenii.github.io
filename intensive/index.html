<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Wordpress-интенсив</title>
    <link rel="stylesheet" href="css/normalize.css">
    <link rel="stylesheet" href="css/fonts.css">
    <link rel="stylesheet" href="css/style.css">
    
</head>

<body>
    <section class="firstScreen">
        <div class="container">
            <div class="logo">
                <img src="icons/logo.png" alt="logo" class="logo_img">
                <div class="logo_text">
                    WordPress интенсив
                </div>    
            </div>
            <h1>
                Создать свой сайт на WordPress 
            </h1>
            <div class="sloganbetweenlines">
                <div class="sloganitems1"> </div>
                <div class="sloganitem">
                    это просто! 
                </div>
                <div class="sloganitems1"></div>
            </div>
            <div class="secondyellowtitle">
                Куда сложнее сделать это правильно!
            </div>
            <p class="everyday">
                Ежедневно в России появляется более 5 000 новых сайтов. <br>
                И только единицы из них становятся популярными и приносят ощутимую прибыль. В чем секрет? В чем формула успеха? Узнайте на предстоящем двухдневном <span class="free">БЕСПЛАТНОМ</span> интенсиве!
            </p>
            <button>
                Записаться на интенсив!
            </button>
            <div class="more">
                <div class="morephrase">
                    узнать больше об интенсиве
                </div>
                <img src="icons/firstBgIcon.png" alt="more">
            </div>
        </div>
    </section>
    <section class="expectations">
            <div class="container">
                <div class="titles">
                    <div class="titleitem1">
                        <div class="title">
                            Что вас ждет в нашем интенсиве?
                        </div>
                        <div class="line1"></div>
                    </div>
                    <div class="titleitem2">
                        2 бесплатных полноценных обучающих вебинара, где:
                    </div>
                </div>

                <div class="expectitems">
                    <div class="expectitem">
                        <img src="icons/secondScreenFirstIcon.png" alt="monitor" class="itempic">
                        <p class="itemtext1">
                            Всего за два бесплатных вебинара мы научим вас, как создать свой сайт на WordPress.
                        </p>
                    </div>
                    <div class="expectitem">
                        <img src="icons/secondScreenSecondIcon.png" alt="monitor" class="itempic">
                        <p class="itemtext1">
                            Покажем, как правильно оптимизировать сайт и сделать его привлекательным для поисковых систем. Только свежая, профессиональная и актуальная информация!
                        </p>
                    </div>
                    <div class="expectitem">
                        <img src="icons/6laptopSecond.png" alt="monitor" class="itempic">
                        <p class="itemtext1">
                            Вы узнаете, что такое «хорошая индексация»,
                            «высокая посещаемость», «стабильный прирост посетителей», «ТОП по запросам» и, что намного важнее, как добиться всех этих показателей!
                        </p>
                    </div>
                </div>
            </div>
    </section>
    
    <section class="opti">
        <div class="container">
            <div class="optitextandimage">
                <img src="img/thirdScreenGirl.jpg" alt="Girl">
                <div class="optitext">
                    <h2 class="optititle">
                        Что такое оптимизация и зачем она нужна?
                    </h2>
                    <p class="optiparagraph">
                        На просторах интернета несчетное количество действующих и даже заброшенных сайтов. Большинство из них совпадает не только по категориям, но и по содержанию.<br><br> 
                        Представьте себе тысячи сайтов, на которых тем или иным образом предоставляется одна и та же информация или продается одна и та же услуга. И попасть на эти сайты можно лишь по тематическим запросам в поисковых системах.<br><br> 
                        Но как вам быть, если ваш сайт похож на сотню или тысячу таких же, а на одной странице гугла или яндекса всего 10 позиций? Как привести посетителей страницы своего ресурса? Именно для этого и нужна оптимизация – для вывода вашего сайта на первую страницу поиска!<br><br>
                        <span class="boldendofopti">Мы научим вас, как сделать ваш сайт привлекательным для поисковых систем и посоветуем, как вывести его на первые страницы, обходя всех конкурентов.</span>
                    </p>
                </div>

            </div>
        </div>
    </section>
    <section class="results">
        <div class="container">
            <div class="titleitem1">
                <div class="title">
                    По окончании обучения Вы сможете!
                </div>
                <div class="line1"></div>
            </div>

            <div class="resultsitems">
                <div class="resultitem">
                    <img src="icons/4screen1icon.png" alt="icon" class="itemicon">
                    <div class="itemtext2">
                        Создать свой сайт или блог
                    </div>
                </div>
                <div class="resultitem">
                    <img src="icons/4screen2icon.png" alt="icon" class="itemicon">
                    <div class="itemtext2">
                        Правильно его оптимизировать
                    </div>
                </div>
                <div class="resultitem">
                    <img src="icons/4screen3icon.png" alt="icon" class="itemicon">
                    <div class="itemtext2">
                        Обеспечить ему медленный, но верный рост в ТОП
                    </div>
                </div>
                <div class="resultitem">
                    <img src="icons/4screen4icon.png" alt="icon" class="itemicon">
                    <div class="itemtext2">
                        Достигнуть стабильного прироста посетителей
                    </div>
                </div>
                <div class="resultitem">
                    <img src="icons/4screen5icon.png" alt="icon" class="itemicon">
                    <div class="itemtext2">
                        Получать ощутимую прибыль
                    </div>
                </div>
            </div>


        </div>
    </section>
    <section class="gift_section">
        <div class="container">
            <img src="icons/gift.png" alt="gift" class="gift">
            <p class="gift_p">
                Для всех участников интенсива предусмотрены бонусы и подарки. Нажимайте на кнопку ниже, принимайте участие в вебинарах и забирайте свой приз по окончании обучения!
            </p>
            <button> 
                Записаться на интенсив!
            </button>
        </div>
    </section>
    <section class="webinar_sec">
        <div class="container">
            <div class="webinar">
                <img src="icons/6laptopFirst.png" alt="webinar_pic" class="webinar_pic">
                <div class="title">
                    Первый вебинар состоится:
                </div>
                <div class="optititle">
                    10 сентября в 20:00 по Москве
                </div>
                <div class="titleitem2">
                    Успейте записаться, количество мест ограничено!
                </div>

            </div>
        </div>
    </section>
    <footer>
        <div class="container">
            <span class="underline">Служба поддержки  |  Политика конфиденциальности</span> <br><br>
            Позвоните нам 8-800-555-01-21
        </div>
    </footer>
</body>

</html>
