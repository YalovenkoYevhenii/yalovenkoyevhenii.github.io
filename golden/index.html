<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Golden</title>
	<link rel="stylesheet" href="css/main.css">
</head>

<body>
	<header>
		<div class="container">
			<div class="heading clearfix">
				<img src="img/logo.png" alt="Golden" class="logo">
				<nav>
					<ul class="menu">
					<li>
						<a href="#">
							Home
						</a>
					</li>
					<li>
						<a href="#">
							Services
						</a>
					</li>
					<li>
						<a href="#">
							Portfolio
						</a>
					</li>
					<li>
						<a href="#">
							About
						</a>
					</li>
					<li>
						<a href="#">
							Contact
						</a>
					</li>

					</ul>
				</nav>
			</div>
			
			<div class="titles">
				<div class="titles_first">
					Welcome To Our Studio!
				</div>

				<h1>
					It’s nice to Meet you
				</h1>
			</div>

			<a href="https://www.youtube.com/" class="button">
				Tell Me More
			</a>

		</div>
	</header>

	<section>
		<div class="container">
			<div class="title">

				<h2>
					Services
				</h2>
				<p>
					Proin iaculis purus consequat sem cure. 
				</p>

			</div>
			<div class="services clearfix">
				<div class="services_item">
					<img src="img/icon1.png" alt="услуга">
					<h3>E-Commerce</h3>
					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>

				</div>

				<div class="services_item">
					<img src="img/icon2.png" alt="услуга">
					<h3>Responsive Web</h3>
					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>

				</div>

				<div class="services_item">
					<img src="img/icon3.png" alt="услуга">
					<h3>Web Security</h3>
					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>

				</div>


			</div>
			
		</div>
	</section>	


	<section id="portfolio">
		<div class="container">
			<div class="title">

				<h2>
					Our portfolio
				</h2>
				<p>
					Proin iaculis purus consequat sem cure. 
				</p>

			</div>
			<div class="works clearfix">
				<img src="img/img1.jpg" alt="работа">
				<img src="img/img2.jpg" alt="работа">
				<img src="img/img3.jpg" alt="работа">
			</div>

		</div>
	</section>


	<section>
		<div class="container">
			<div class="title">

				<h2>
					About Us
				</h2>
				<p>
					Proin iaculis purus consequat sem cure. 
				</p>

			</div>
			<div class="clearfix">
				<div class="achivements clearfix">
					<div class="h4p clearfix">
						<h4>
							<span> 
								July 2010 
							</span><br>
							Our Humble Beginnings
						</h4>
						<p>
						Proin iaculis purus consequat sem cure <br> 
						digni ssim. Donec porttitora entum suscipit <br> 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni <br>
						ssim. Donec porttitora entum suscipit.	
						</p>
					</div>
					<img src="img/about1.jpg" alt="достижение">
				</div>
					
				<div class="achivements1 clearfix">
					<div class="h4p1 clearfix">
						<h4>
							<span> January 2011 </span><br>
							Facing Startup Battles
						</h4>
						<p>
						Proin iaculis purus consequat sem cure <br> 
						digni ssim. Donec porttitora entum suscipit <br> 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni <br>
						ssim. Donec porttitora entum suscipit.	
						</p>
					</div>
					<img src="img/about2.jpg" alt="достижение">
				</div>
				
				<div class="achivements clearfix">
					<div class="h4p clearfix">
						<h4>
							<span> January 2011 </span><br>
							Enter The Dark Days
						</h4>
						<p>
						Proin iaculis purus consequat sem cure <br> 
						digni ssim. Donec porttitora entum suscipit <br> 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni <br>
						ssim. Donec porttitora entum suscipit.	
						</p>
					</div>
					<img src="img/about3.jpg" alt="достижение">
				</div>

				<div class="achivements1 clearfix">
					<div class="h4p1 clearfix">
						<h4>
							<span> January 2011 </span><br>
							Our Triumph
						</h4>
						<p>
						Proin iaculis purus consequat sem cure <br> 
						digni ssim. Donec porttitora entum suscipit <br> 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni <br>
						ssim. Donec porttitora entum suscipit.	
						</p>
					</div>
					<img src="img/about4.jpg" alt="достижение">
				</div>
			</div>

		</div>
	</section>

	<footer>
		<div class="container ">
			<h5>
				© Copyright 2014 FreebiesXpress.com
			</h5>


			 
		</div>
	</footer>



</body>



</html>
































